libs/shared/package.json

```diff
@@ -25,6 +25,18 @@
   "files": [
     "dist/"
   ],
+  "exports": {
+    ".": {
+      "require": "./dist/cjs/index.js",
+      "import": "./dist/esm/index.js",
+      "types": "./dist/esm/index.d.js"
+    },
+    "./utils": {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** What is the benefit for introducing another barrel entry point for shared vs just importing from root?‏

Reply:
It's a new pattern to avoid one level of barrel files since node now support subpath imports. When used extensively, Barrel files affect [performance](https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/).

Of course, just one change won't have any impact, I am mostly testing whether the pattern can be applied moving forward.